Sources/AppsFlyerInstance.swift

```diff
@@ -61,31 +72,78 @@ public class AppsFlyerInstance: NSObject, AppsFlyerCommand {
+        if let disableAdTracking = settings[AppsFlyerConstants.Configuration.disableAdTracking.rawValue] as? Bool {
             appsFlyer.disableAdvertisingIdentifier = disableAdTracking
             appsFlyer.disableIDFVCollection = disableAdTracking
         }
-        if let disableAppleAdTracking = settings[AppsFlyerConstants.Configuration.disableAppleAdTracking] as? Bool {
+        if let disableAppleAdTracking = settings[AppsFlyerConstants.Configuration.disableAppleAdTracking.rawValue] as? Bool {
             appsFlyer.disableSKAdNetwork = disableAppleAdTracking
         }
-        if let minTimeBetweenSessions = settings[AppsFlyerConstants.Configuration.minTimeBetweenSessions] as? Int {
+        if let disableAppleAdsAttribution = settings[AppsFlyerConstants.Configuration.disableAppleAdsAttribution.rawValue] as? Bool {
+            appsFlyer.disableAppleAdsAttribution = disableAppleAdsAttribution
+        }
+        if let disableCollectASA = settings[AppsFlyerConstants.Configuration.disableCollectASA.rawValue] as? Bool {
+            appsFlyer.disableCollectASA = disableCollectASA
+        }
+        if let minTimeBetweenSessions = settings[AppsFlyerConstants.Configuration.minTimeBetweenSessions.rawValue] as? Int {
             appsFlyer.minTimeBetweenSessions = UInt(minTimeBetweenSessions)
         }
-        if let anonymizeUser = settings[AppsFlyerConstants.Configuration.anonymizeUser] as? Bool {
+        if let anonymizeUser = settings[AppsFlyerConstants.Configuration.anonymizeUser.rawValue] as? Bool {
             appsFlyer.anonymizeUser = anonymizeUser
         }
-        if let shouldCollectDeviceName = settings[AppsFlyerConstants.Configuration.collectDeviceName] as? Bool {
+        if let shouldCollectDeviceName = settings[AppsFlyerConstants.Configuration.collectDeviceName.rawValue] as? Bool {
             appsFlyer.shouldCollectDeviceName = shouldCollectDeviceName
         }
-        if let customData = settings[AppsFlyerConstants.Configuration.customData] as? [AnyHashable: Any] {
+        if let customData = settings[AppsFlyerConstants.Configuration.customData.rawValue] as? [AnyHashable: Any] {
             appsFlyer.customData = customData
         }
+        if let useUninstallSandbox = settings[AppsFlyerConstants.Configuration.useUninstallSandbox.rawValue] as? Bool {
+            appsFlyer.useUninstallSandbox = useUninstallSandbox
+        }
+        if let enableTCFDataCollection = settings[AppsFlyerConstants.Configuration.enableTCFDataCollection.rawValue] as? Bool {
+            appsFlyer.enableTCFDataCollection(enableTCFDataCollection)
+        }
+        if let appInviteOneLinkID = settings[AppsFlyerConstants.Configuration.appInviteOneLinkID.rawValue] as? String {
+            appsFlyer.appInviteOneLinkID = appInviteOneLinkID
+        }
+        if let deepLinkTimeout = settings[AppsFlyerConstants.Configuration.deepLinkTimeout.rawValue] as? Int {
```

Comment:
should check that deepLinkTimeout is greater than 0 or this could crash

```suggestion
        if let deepLinkTimeout = settings[AppsFlyerConstants.Configuration.deepLinkTimeout.rawValue] as? Int, 
        deepLinkTimeout > 0 {
```